src/webgpu/api/operation/rendering/blending.spec.ts

```diff
@@ -372,6 +436,67 @@ g.test('formats')
     t.expectOK(result);
   });
 
+g.test('default_blend_color,initial_blend_color')
```

Comment:
At some point we changed the terminology in the spec from "blend color" to "blend constant", which has left some old code a bit confusing (and now I realize what one of the tests in #1976 is about...). Let's apply that rename to the changes in this PR, so that it will be clearer what it's testing.

Reply:
ok, I rename `blend_color` to `blend_constant`.